API/projectClass/projectClass.m

```diff
@@ -773,21 +773,19 @@
             
             % Make a different allowed list depending on whether 
             % it is custom or layers
-            ModelType = obj.ModelType;
+            modelType = lower(obj.modelType);
             
-            if ~strcmpi(ModelType,{'custom layers','custom xy'})
+            if ~strcmpi(modelType, {'custom layers','custom xy'})
                 % Standard Layers
                 allowedValues = obj.layers.getLayersNames();
-                ModelType = 'standard';
                 inputVals = inputVals{:};
             else
                 % Custom models
                 allowedValues = obj.customFile.getCustomNames();
-                ModelType = 'custom';
             end
             
             % Call the setContrastModel method
-            obj.contrasts.setContrastModel(thisContrast,ModelType,allowedValues,inputVals);
+            obj.contrasts.setContrastModel(thisContrast, modelType, allowedValues, inputVals);
```

Comment:
The routine setContrastModel expects the model type to be either "standard" or "custom". This needs to be updated to match the removal of those labels here.